dotcom-rendering/src/web/layouts/Layout.stories.tsx

```diff
@@ -163,13 +163,20 @@ for (const [displayName] of Object.entries(ArticleDisplay)) {
 	}
 }
 
-storiesOf(`Layouts/Liveblog`, module).add('With no key events', () => {
-	return (
-		<HydratedLayout
-			ServerCAPI={{
-				...Fixtures.LiveBlog,
-				keyEvents: [],
-			}}
-		/>
-	);
-});
+storiesOf(`Layouts/Liveblog`, module)
+	.add('With no key events', () => {
+		return (
+			<HydratedLayout
+				ServerCAPI={{
+					...Fixtures.LiveBlog,
+					keyEvents: [],
+				}}
+			/>
+		);
+	})
+	.addParameters({
+		chromatic: {
+			diffThreshold: 0.2,
```

Comment:
What does diffThreshhold do?

Reply:
Here's an example of where this has been useful in the past @OllysCoding https://github.com/guardian/image-rendering/pull/137

Reply:
Great, thank you :)